dependency.lic

```diff
@@ -770,8 +770,46 @@ class SetupFiles
     boxes = @base_files['base-items.yaml'][:box_nouns]
     gems = @base_files['base-items.yaml'][:gem_nouns]
     scrolls = @base_files['base-items.yaml'][:scroll_nouns]
-    s.lootables = (base_lootables + boxes + gems + scrolls + s.loot_additions - s.loot_subtractions).uniq
+    spells = @base_files['base-spells.yaml'][:spell_data] 
 
+    s.lootables = (base_lootables + boxes + gems + scrolls + s.loot_additions - s.loot_subtractions).uniq
+    
+    s.offensive_spells
+      .select { |spell| spells[spell['name']] }
+      .each_with_index do |spell, i|
+        s.offensive_spells[i] = spells[spell['name']].merge(spell)
+      end
```

Comment:
We could probably replace these with
```
 s.offensive_spells.map!{ |spell| spells[spell['name']] ? spells[spell['name']].merge(spell) : spell  }
```

And I'd still be tempted to pull out that inner ternary to a helper method like `merge_spell_defaults(spells, spell)`